packages/core/src/oidc/init.ts

```diff
@@ -89,6 +99,19 @@ export default async function initOidc(app: Koa): Promise<Provider> {
         },
       };
     },
+    ttl: {
+      /**
+       * [OIDC Provider Default Settings](https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#ttl)
+       */
+      IdToken: (ctx, token, client) => {
+        const { idTokenTtl } = client.metadata();
+        return idTokenTtl ?? 60 * 60;
+      },
+      RefreshToken: (ctx, token, client) => {
+        const { refreshTokenTtl } = client.metadata();
+        return refreshTokenTtl ?? 14 * 24 * 60 * 60;
```

Comment:
I think it's better to avoid magic numbers, we can use constant with meaningful words instead.

Reply:
@intsilence please check